WrapImage/nifti_wrapper.py

```diff
@@ -102,7 +103,7 @@ def loop_over_first_n_minus_1_dimensions(arr):
         Dp_image = []
         D_image = []
 
-        for idx, view in loop_over_first_n_minus_1_dimensions(data):
+        for idx, view in tqdm(loop_over_first_n_minus_1_dimensions(data), desc=f"{args.algorithm} is fitting", dynamic_ncols=True, total=702464):
```

Comment:
Is this necessary? `total=702464` We can't hard code any numbers because we don't know the image size. If it's needed could we just extract it from the image itself?

Reply:
Was there an update to this? I don't see a change.